src/datasources/library.datasource.js

```diff
@@ -101,42 +83,16 @@ const age = () => {
  * @returns {Promise<(*&{branchShortName, branchName, openingHours, illOrderReceiptText})[]>}
  */
 async function doRequest() {
-  let result;
-
-  // get active libraries first (librarStatus:"AKTIVE")
-  const active = await post("AKTIVE");
-  // set active status on active libraries
-  active.forEach((del) => (del.libraryStatus = "AKTIVE"));
-  // now get deleted libraries (libraryStatus: "SLETTET")
-  const deleted = await post("SLETTET");
-  // set deleted status on deleted libraries
-  deleted.forEach((del) => (del.libraryStatus = "SLETTET"));
-  // now get invisible libraries (libraryStatus:"USYNLIG")
-  // invisible libraries are also included in active
-  const invisible = await post("USYNLIG");
-  // set invisible status on invisible libraries
-  invisible.forEach((del) => (del.libraryStatus = "USYNLIG"));
-  // merge arrays to get ALL libraries with libraryStatus set on relevant ones
-  const alllibrarieswithstatus = [...invisible, ...active, ...deleted];
-  // filter out duplicates
-  const uniqueLibraries = uniqBy(alllibrarieswithstatus, "branchId");
+  const libraries = await get();
 
-  result = uniqueLibraries.map((branch) => {
-    return {
-      ...branch,
-      branchName:
-        branch.branchName && branch.branchName.map((entry) => entry.value),
-      branchShortName:
-        branch.branchShortName &&
-        branch.branchShortName.map((entry) => entry.value),
-      openingHours:
-        branch.openingHours && branch.openingHours.map((entry) => entry.value),
-      illOrderReceiptText:
-        branch.illOrderReceiptText &&
-        branch.illOrderReceiptText.map((entry) => entry.value),
-    };
-  });
-  return result;
+  return libraries.map((branch) => ({
+    ...branch,
+    agencyName: branch.agencyNames?.join(" "),
```

Comment:
Der ligger allerede en "agencyName" på endpointet, som (nogengange) er rigtigt. "agencyNames" indeholder nogengange alle branches og nogengange også alias'er for branches... Nogengange indeholder det bl.a. navnet på agenciet... 

Er der ikke noget galt med `allLibraries`?